internal/util/identifiers.go

```diff
@@ -51,13 +51,22 @@ func GenerateLobbyCode(ctx context.Context) string {
 	return strconv.FormatInt(randInt63(), 36)
 }
 
-func GenerateShortLobbyCode(ctx context.Context) string {
+func GenerateShortLobbyCode(ctx context.Context, chars int) string {
 	randIntn := rand.Intn
 	if os.Getenv("ENV") == "test" {
 		randIntn = deterministicRand.Intn
 	}
 
 	numbers := []string{"2", "3", "4", "5", "6", "7", "8", "9"}
 	alphabet := []string{"A", "B", "C", "D", "E", "F", "G", "H", "J", "K", "M", "N", "P", "R", "S", "T", "V", "W", "X", "Y", "Z"}
-	return numbers[randIntn(len(numbers))] + numbers[randIntn(len(numbers))] + alphabet[randIntn(len(alphabet))] + alphabet[randIntn(len(alphabet))]
+
+	ss := make([]string, chars)
+	for i := range chars {
+		if i/2%2 == 0 {
```

Comment:
`i/2%2` is very obscure. 😅 I ran it in a Python interpreter to see what it is, which made me more confused. Turns out, it's different in Go and Python. I think a quick comment will fix this. 😇 <small>(And maybe a unittest?)</small>

Reply:
Yeah I removed this now. We now just use the whole alphabet for all 4 characters. Otherwise the amount of possible lobbies was too low.

Reply:
We have had multiple games now with the issue if running out of possible lobbies.